packages/apps/src/Endpoints/index.tsx

```diff
@@ -216,21 +225,29 @@ function Endpoints ({ className = '', offset, onClose }: Props): React.ReactElem
   const _onApply = useCallback(
     (): void => {
       settings.set({ ...(settings.get()), apiUrl });
-
-      window.location.assign(`${window.location.origin}${window.location.pathname}?rpc=${encodeURIComponent(apiUrl)}${window.location.hash}`);
+      window.location.assign(`${window.location.origin}${window.location.pathname}${getApiType(apiUrl).type === 'substrate-connect' as EndpointType ? '?sc=' : '?rpc='}${encodeURIComponent(apiUrl)}${window.location.hash}`);
```

Comment:
One more `as EndpointType`, something is weird with the typing somewhere...

Reply:
Will remove from everywhere